Codecademy/gamut
# fix(Badge): color of secondary badges

### Overview


Fixes the color of secondary variant badges to match figma


### PR Checklist

- [x] Related to designs: [badges](https://www.figma.com/file/ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?type=design&node-id=29959-39721&mode=design&t=lUXIK9CoJEM1apbf-0)
- [x] Related to JIRA ticket: [web-2485]

#### Testing Instructions

* Visit the Badge page 
* See the new colors for the secondary variant on light + dark mode
* 
#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Mono       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/4256)  | [Portal Env](n/a)   |
